metricbeat/module/system/diskio/diskio.go

```diff
@@ -21,20 +21,29 @@ func init() {
 type MetricSet struct {
 	mb.BaseMetricSet
 	statistics *DiskIOStat
+	names      []string
 }
 
 // New is a mb.MetricSetFactory that returns a new MetricSet.
 func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
-	ms := &MetricSet{
+	config := struct {
+		Names []string `config:"diskio.include_devices"`
```

Comment:
Can we name this variable `IncludeDevices`. This makes it easier to follow the code I think.

Reply:
+1 would make sense

Reply:
@stefws Not sure I get your comment above?

Reply:
@ruflin I seconded your comment on the more sensible name for the variable and since this is a code change it means that this feature isn't currently available yet (as in v.5.6.5) which I mistakenly thought was whats iahmedov ment when he said: 
'since psutil/disk.IOCounters (which diskio uses) supports filtering by exact match names'